components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -1124,4 +1125,42 @@ public static boolean isUseVerifyClaimEnabled() {
         return Boolean.parseBoolean(IdentityUtil.getProperty
                 (IdentityRecoveryConstants.ConnectorConfig.USE_VERIFY_CLAIM_ON_UPDATE));
     }
+
+    /**
+     * Trigger recovery event.
+     *
+     * @param map       Property map.
+     * @param eventName Event Name.
+     * @param OTP       OTP.
```

Comment:
I think OTP refers to the confirmation code right? Shall we improve the variable identifier?